.github/local-development.md

```diff
@@ -0,0 +1,107 @@
+# Local Development
+
+This document gives tips and tricks on how to create the Renovate documentation website locally to add features or fix bugs.
+You can improve this documentation by opening a pull request.
+For example, if you think anything is unclear, or you think something needs to be added, open a pull request!
+
+## Installation
+
+### Prerequisites
+
+You need the following dependencies for local development:
+
+- Git
+- Node.js `^12.13.0 || >=14.15.0`
+- Yarn `^1.17.0`
+- Python `^3.8`
```

Comment:
```suggestion
- Python `^3.9`
```

Do we want to use the latest stable Python release?

Reply:
Sure latest